### PR TITLE
Added t2t-vit-14 model

### PR DESCRIPTION
**Pull request by Anna-Mironova**
add t2t-vit-14 image classification public model

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
@Anna-Mironova pre-commits passed, can we change from Draft status?

File: models/public/t2t-vit-14/model.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  - name: 81.5_T2T_ViT_14.pth
+    size: 86212823
+    checksum: 98c8ae06a1e9997f3e83ad2dfbac49985bf7bd7723eb64e4b8462338163be02657998f0bd9f8997367417003caeaf7a1
+    source: https://github.com/yitu-opensource/T2T-ViT/releases/download/main/81.5_T2T_ViT_14.pth.tar
+postprocessing:
+  - $type: unpack_archive
+    format: zip
+    file: timm-0.5.4-py3-none-any.whl
+  - $type: regex_replace
+    file: models/t2t_vit.py
```
Comment (eaidova) [inline models/public/t2t-vit-14/model.yml:54]: please stay comment why you replace token performer with own impleentation

**Review by eaidova — APPROVED** [review]

File: models/public/t2t-vit-14/model.yml (hunk 1 above)
Comment (Anna-Mironova) [reply models/public/t2t-vit-14/model.yml:58]: Done

Comment (eaidova) [reply models/public/t2t-vit-14/model.yml:58]: hmm... is it not better to increase opset_version instead?

Comment (Anna-Mironova) [reply models/public/t2t-vit-14/model.yml]: Done

**Pull request merged**